OOP Fundamentals and C#/09 - CreationalPatterns/Factory/FactoryMethod/Truck.cs

```diff
@@ -7,9 +8,23 @@ public class Truck : ITransport
         public string Name { get; set; }
```

Comment:
Please remove these properties after when you use abstract class instead of interface